### PR TITLE
GB-6683 - Show a link to the dashboard when running create

### PR DESCRIPTION
**Pull request by yoav-lavi**
# Description

## Features

- Shows a dashboard link when creating a project

# Type of change

- [ ] 💔 Breaking
- [x] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/crates/cli/src/output/report.rs [hunk 1]
```diff
@@ -483,12 +483,11 @@ pub fn list_environment_variables(environment_variables: Vec<EnvironmentVariable
...
-pub fn create_success(name: &str, urls: &[String]) {
+pub fn create_success(name: &str, urls: &[String], account_slug: &str) {
     watercolor::output!("\n✨ {name} was successfully created!\n", @BrightBlue);
-    watercolor::output!("Endpoints:", @BrightBlue);
-    for url in urls {
-        watercolor::output!("- https://{url}", @BrightBlue);
-    }
+    urls.first()
+        .map(|url| watercolor::output!("Endpoint: https://{url}", @BrightBlue));
+    watercolor::output!("Dashboard: https://app.grafbase.com/{account_slug}/{name}/branches/main", @BrightBlue);
```
[the remark this answers is not in the document]
Comment (yoav-lavi) [reply cli/crates/cli/src/output/report.rs:490]: Not sure what you mean by a dev build in this context, this is `create` so nothing exists except prod

[the remark this answers is not in the document]
Comment (fbjork) [reply cli/crates/cli/src/output/report.rs:491]: builds that don't point to production shouldn't open the prod dashboard url.

[the remark this answers is not in the document]
Comment (yoav-lavi) [reply cli/crates/cli/src/output/report.rs:491]: Oh, you mean if a project is self hosted or on our `.dev`, adding

[the remark this answers is not in the document]
Comment (yoav-lavi) [reply cli/crates/cli/src/output/report.rs]: Done, added `GRAFBASE_DASHBOARD_URL`

File: cli/crates/backend/src/api/consts.rs [hunk 2]
```diff
@@ -7,6 +7,7 @@ pub const TAR_CONTENT_TYPE: &str = "application/x-tar";
 pub const GRAFBASE_ACCESS_TOKEN_ENV_VAR: &str = "GRAFBASE_ACCESS_TOKEN";
 
 const API_URL: &str = "https://api.grafbase.com/graphql";
+const DASHBOARD_URL: &str = "https://app.grafbase.com";
```
Comment (yoav-lavi) [inline cli/crates/backend/src/api/consts.rs:10]: @fbjork IIRC we left the auth URL as is on purpose but we could use this for that as well

Comment (yoav-lavi) [reply cli/crates/backend/src/api/consts.rs:10]: @fbjork any opinion here?

**Review by obmarg — APPROVED** [review]

**Pull request merged**